kythe/go/platform/kzip/kzip.go

```diff
@@ -197,6 +197,7 @@ func NewReader(r io.ReaderAt, size int64) (*Reader, error) {
 	if err != nil {
 		return nil, err
 	}
+	log.Printf("Reading kzip with %v encoding", pref)
```

Comment:
This seems like log spam.  Maybe expose the encoding as a method and log at the point of interest?

Reply:
done